agentstack/generation/gen_utils.py

```diff
@@ -66,3 +71,72 @@ def string_in_file(file_path: str, str_to_match: str) -> bool:
         file_content = file.read()
         return str_to_match in file_content
 
+
+def _framework_filename(framework: str, path: str = ''):
+    if framework == 'crewai':
+        return f'{path}src/crew.py'
+
+    print(term_color(f'Unknown framework: {framework}', 'red'))
+    sys.exit(1)
+
+
+class CrewComponentType(str, Enum):
+    AGENT = "agent"
+    TASK = "task"
+
+
+def get_crew_components(
+        framework: str = 'crewai',
+        component_type: Optional[Union[CrewComponentType, List[CrewComponentType]]] = None,
+        path: str = ''
+) -> dict[str, List[str]]:
+    """
```

Comment:
Also your method docstrings are legit

Reply:
lol ill tell copiliot you said thank you